src/fragments/lib/auth/js/set-up-auth.mdx

```diff
@@ -1,25 +1,23 @@
-
-## Prerequisites
+In this guide, you will learn how to set up Amplify **Auth**. This includes setting up and connecting your backend resources, determining your integration path, and enabling sign-up, sign-in, and sign-out with Authenticator UI component. We will also provide more context on how resources are managed and created with Amplify to help you make decisions, and understand any long-term impact of those decisions.
 
 Before you begin, you will need:
 - [Node.js](https://nodejs.org/) v14.x or later
 - [npm](https://www.npmjs.com/) v6.14.4 or later
 - [git](https://git-scm.com/) v2.14.1 or later
-- A front-end app
-- If you are want Amplify to set up and manage your backend resources you need to [install and configure](/lib/project-setup/prereq/q/platform/js/) the Amplify CLI. Make sure to also created a new amplify project using `amplify init` in your terminal, or pull in an existing amplify project to your frontend app using `amplify pull`.
+- A frontend app
+- If you are want Amplify to set up and manage your backend resources you need to [install and configure](/lib/project-setup/prereq/q/platform/js/) the Amplify CLI. Make sure to also create a new amplify project using `amplify init` in your terminal, or pull in an existing amplify project to your frontend app by using `amplify pull`.
 
 ## Set up and connect backend resources
 
-We will also guide you on the paths to integrating Amplify Auth and how to set up and integrate your backend resources, and how to connect to these resources in your frontend app to build authentication features.
+We will review the paths to integrate Amplify Auth before you set up and integrate your backend resources and connect these resources in your frontend app to build authentication features.
 
 ### Decide how to create and manage your backend resources
 
 You can create and manage your Authentication resources with Amplify by using the Amplify CLI, Amplify Studio, or manage them yourself with tools such as CDK and CloudFormation. The path we recommend is via the Amplify CLI, which allows you to create new authentication resources or import existing ones. However, you can also use the Amplify Studio console to configure or use existing resources and directly connect them to your application using the Amplify Libraries. These tools will help you with creating and managing your resources.
 
-With Amplify Auth, you can use a username and password as an authentication method, use a social provider Sign In with Google or Sign in With Apple, or create a fully custom authentication flow.
+With Amplify Auth, you can use a username and password as an authentication method, use a social provider "Sign In with Google" or "Sign in With Apple," or create a fully custom authentication flow.
```

Comment:
With Amplify Auth, you can use a username and password as an authentication method, use a social provider **such as** "Sign In with Google" or "Sign in With Apple," or create a fully custom authentication flow.

Reply:
Updated